nowide/CMakeLists.txt

```diff
@@ -12,18 +12,19 @@ if (BUILDING_LEATHERMAN)
     set(BOOST_NOWIDE_SKIP_INSTALL ON CACHE BOOL "Disable installing Boost.Nowide")
 
     # have to specify a bindir since this isn't actually a subdirectory of us
-    add_subdirectory("../vendor/boost-nowide" "../vendor/boost-nowide")
+    set(srcdir "../vendor/boost-nowide")
+    set(bindir "${CMAKE_CURRENT_BINARY_DIR}/${srcdir}")
```

Comment:
If we're making bindir based on `CMAKE_CURRENT_BINARY_DIR` we should probably go all in and make srcdir depend on `CMAKE_CURRENT_SOURCE_DIR`